rocky/katalogus/client.py

```diff
@@ -47,8 +47,13 @@ class Plugin(BaseModel):
     created: str | None = None
     description: str | None = None
     related: list[str] = Field(default_factory=list)
-    enabled: bool
     type: str
+    # TODO: this is the only field making a Plugin organization-specific. If we could separate the use of the enabled
+    #  field from other uses, we would be able to drop the organization_code as an argument in a lot of places, hence
+    #  simplifying the usage of the KATalogus for installation-wide operations (as plugins are not specific to an
+    #  organization). One could argue that having this field here should mean we need an organization field as well to
+    #  make sense out of it: for which organization is this plugin in fact enabled?
+    enabled: bool
```

Comment:
Looking at the future, I can see two types of plugins. Those (base plugins) listed on the remote repos (orga agnostic, not enabled), and those copied to a local organizations XTDB namespace, in which they can be enabled, forked, configured etc.

By doing this, it also becomes easier to manage variants and ownership of these installed plugins per organization, as deleting/changing them only affects the local organization instead of all organizations in the same install.

Sharing the local changes back to other organizations (local or remote) can be done by providing the base plugins either to the other organizations or by uploading them to the remote or public repos.

Reply:
I think at least we should try to work with "base plugins" while/where we can since managing cross-organization plugins will be significantly more work